kmm/kmm-post-data/src/commonMain/kotlin/com/gchristov/newsfeed/kmmpostdata/model/Post.kt

```diff
@@ -10,6 +10,7 @@ data class DecoratedPost(
     // Additional properties
     val date: Instant,
     val favouriteTimestamp: Long? = null, // Will be set later if post has been added to favourites
+    val readingTimeMinutes: Int = 1
```

Comment:
Nitpick: 1 is okay but we could have this as `Int? = null` to indicate that it has not been calculated yet, similar to the `favouriteTimestamp`

Reply:
applied